programmers/mock-test.js

```diff
@@ -0,0 +1,30 @@
+function solution(answers) {
+    let collect1 = 0;
+    let collect2 = 0;
+    let collect3 = 0;
+    for (let i = 0; i < answers.length; i++) {
+        if ( 수포자1[i % 5] === answers[i]) {
+            collect1 += 1;
+        }
+        if ( 수포자2[i % 8] === answers[i]) {
+            collect2 += 1;
+        }
+        if ( 수포자3[i % 10] === answers[i]) {
+            collect3 += 1;
+        }
+    }
+    const collects = [collect1, collect2, collect3];
+    const biggest = Math.max(...collects);
+    const results = [];
+    for (let j = 0 ; j < 3; j++) {
+        if (collects[j] === biggest) {
+            results.push(j+1);
+        }   
+    }
+    return results;
```

Comment:
같은 문제 수를 맞춘 경우도 있으니 if문을 3개 써줍니다.
```suggestion
    if (count[0] === max) {
        answer.push(1);
    }
    if (count[1] === max) {
        answer.push(2);
    }
    if (count[2] === max) {
        answer.push(3);
    }
    return answer;
```